src/formatter.ts

```diff
@@ -0,0 +1,156 @@
+import { blueBright, dim, red, reset, underline, yellow } from 'chalk';
+import { ESLint } from 'eslint';
+import stripAnsi from 'strip-ansi';
+import table from 'text-table';
+import { TodoResultMessage } from 'types/estlint-type-extension';
+
+function pluralize(word: string, count: number): string {
+  return count === 1 ? word : `${word}s`;
+}
+
+export function formatter(
+  results: ESLint.LintResult[],
+  shouldIncludeTodo?: boolean
```

Comment:
I'd prefer we don't use a boolean here, as it makes it hard to read when viewing the callsite.

```ts
let result = formatter(results);

//or 

let result = formatter(results, true); // what does true represent?
```

vs.

```ts
let result = formatter(results, { includeTodo: true });
```

You can default it to false in a default object literal.

Reply:
Ooh, that's a good call. Done.